tests/functional/tests.py

```diff
@@ -1327,7 +1327,7 @@ def test_presigned_get_object_expiry_5sec(client, log_output):
 
         presigned_get_object_url = client.presigned_get_object(bucket_name,
                                                                object_name,
-                                                               timedelta(seconds=5))
+                                                               timedelta(seconds=120))
```

Comment:
Function names here says `test_presigned_get_object_expiry_5sec`, that should be changed. Other functions where we have changed the code should also have their names changed to reflect the changes made.

Reply:
Done

Reply:
Done